lib/WUI/netif_settings.h

```diff
@@ -24,6 +24,16 @@ typedef enum {
     ETH_NETIF_UP,   // ETH interface is UP
 } ETH_STATUS_t;
 
+typedef enum {
+    WUI_ETH_LINK_DOWN, // ETH cable is unplugged
+    WUI_ETH_LINK_UP,   // ETH cable is plugged
+} WUI_ETH_LINK_STATUS_t;
+
+typedef enum {
+    WUI_ETH_NETIF_DOWN, // ETH interface is down
+    WUI_ETH_NETIF_UP,   // ETH interface is up
+} WUI_ETH_NETIF_STATUS_t;
+
 typedef struct {
```

Comment:
pack to 32 bits (don't start with 8 bits)

Reply:
i do not understand, you probably meant that compiler would allocate only a byte in this case? May be I have not understood the discussions/ some compiler settings  in our project?